pkg/agent/nodeportlocal/k8s/annotations.go

```diff
@@ -41,16 +42,20 @@ func toJSON(serialize interface{}) string {
 	return string(jsonMarshalled)
 }
 
-func isNodePortInAnnotation(s []NPLAnnotation, nodeport int) bool {
+func isNodePortInAnnotation(s []NPLAnnotation, nodeport, cport int) bool {
 	for _, i := range s {
 		if i.NodePort == nodeport {
-			return true
+			if i.PodPort == cport {
+				return true
+			}
 		}
 	}
 	return false
 }
 
-func assignPodAnnotation(pod *corev1.Pod, nodeIP string, containerPort, nodePort int) {
+// AssignPodAnnotation creates an annotation for a Pod with the assigned nodePort, and
+// updates it into the Pod, it returns true if an update to the Pod is required
```

Comment:
into the Pod, -> "into the Pod."

Add "." at the end of the sentence: "is required."

Reply:
Fixed.